internal/app/grpcserver/impl.go

```diff
@@ -136,7 +139,8 @@ func asAny(msg proto.Message) (*anypb.Any, error) {
 	if err != nil {
 		return nil, status.Errorf(
 			codes.Internal,
-			fmt.Sprintf("unable to convert message: %s", err.Error()),
+			"unable to convert message: %s",
```

Comment:
aside: It's quite annoying the `status.Errorf` doesn't actually support the "w" directive, for wrapping errors :/

Reply:
Had the same thought. Makes no sense.